### DISCUSSION TITLE
Any update on the new version?

### DISCUSSION THREAD

**Discussion by erichaydel** [collerek/ormar#992] [General]
Hey collerek! Any thoughts on releasing a new version with the speedup fixes? It would be nice to have a 0.12.1 or 0.13.0 to pin to for my projects.
@collerek

**Comment by collerek** [comment]
Hi, sorry was buried in life a little, wanted to include more fixes but I guess will release it as is to not delay it further :)

**Reply by erichaydel** [reply]
No worries! Hope all is well
